addons/send/send.lua

```diff
@@ -4,10 +4,24 @@ _addon.command = 'send'
 _addon.author = 'Byrth'
 
 windower.register_event('addon command',function (...)
-    local term = table.concat({...}, ' ')
+	local term = table.concat({...}, ' ')
```

Comment:
Please fix the indentation in the entire file to be 4 spaces. This is not your fault, the file was inconsistent before, I'd just like to ask you to do it since you're modifying it.